docs/src/_cloud/onboarding.md

```diff
@@ -93,21 +81,22 @@ pipx reinstall meltano-cloud-cli
 
 The above commands will install a `meltano-cloud` CLI command into your workstation.
 
-### Step 3: User creation and authorization
+### Step 2: User creation and authorization
 
 First, create your user account by logging in via: `meltano-cloud login`.
 
 The login command will open a browser window which you can use to create your account. In the Beta, your identification and authorization will be driven by your GitHub login identity. No Meltano-specific passwords or usernames are needed, and Meltano Cloud does not have access to your personal GitHub credentials.
 
-After you create your account, within one business day, your account will be authorized according to the onboarding information you'll see be able to access your project within the CLI.
+While Meltano Cloud is in Beta, it may take up to one business day for your user to be fully provisioned and added to your Meltano Cloud organization. You'll be notified via email once your newly created user has been added to your organization, at which time you can logout via `meltano-cloud logout` and then log back in to have full CLI access to your organization's Meltano Cloud resources.
```

Comment:
```suggestion
```
I think we can delete this paragraph, right? They won't have to logout and login.

Reply:
Good callout, I didn't realize this but just confirmed with @magreenbaum that this is the case. Removing.